pam/integration-tests/vhs-helpers_test.go

```diff
@@ -302,28 +313,50 @@ func (td tapeData) Output() string {
 	return txt
 }
 
-func (td tapeData) ExpectedOutput(t *testing.T, outputDir string) string {
+func checkDataRace(t *testing.T, raceLog string) {
 	t.Helper()
 
-	outPath := filepath.Join(outputDir, td.Output())
-	out, err := os.ReadFile(outPath)
-	require.NoError(t, err, "Could not read output file of tape %q (%s)", td.Name, outPath)
-	got := string(out)
+	if !testutils.IsRace() || raceLog == "" {
+		return
+	}
```

Comment:
That means we ignore `go test` exiting with a non-zero exit code if the race logfile is empty, right? Is that what we want?

Reply:
I think it also means we ignore any error if `go test` is not run with `-race`

Reply:
However, we also fail on that line if we "ignored" the error in `checkDataRace`, right? What's the purpose of `checkDataRace` then?

Reply:
No, in case we detected the race, we're skiping the test... And `Skip` implies `skipNow`, thus no further checks will happen and we'll just go through the cleanups.

Reply:
I've been taught here that using `else` is evil, thus the convolution :) but if you like it better...

Reply:
> I've been taught here that using else is evil

As a rule of thumb I agree, but if it hurts clarity (like it in this case) I very much prefer clarity

Reply:
I've just dropped the logging, since it would happen still via skip, FWIW... And CI will include it.